CHANGELOG.md

```diff
@@ -6,6 +6,26 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [4.2.0]
+### Added
+- Added `EncryptionKey` type to hold a `CryptoKey` along with its derivation parameters ([#49](https://github.com/MetaMask/browser-passworder/pull/49))
```

Comment:
Nit: Perhaps we could group these entries under a "Support key derivation options" entry? Or something along those lines. It's great that we have all of the details well explained here, but the high-level change isn't super clear

Reply:
I guess we could move the "change" entries under this larger point as well. Or have another group entry under "changed", something like "Add optional parameters and properties to support custom derivation options"